tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/OrdersBuilder.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders
+{
+    internal sealed class OrdersBuilder
+    {
+        private readonly Order _order;
+
+        private OrdersBuilder()
+        {
+            _order = new Order()
+            {
+                OrderId = "C000014-01",
+                Description = "Some Description",
+                OrganisationId = Guid.NewGuid(),
+                Created = DateTime.UtcNow,
+                LastUpdated = DateTime.UtcNow,
+                LastUpdatedBy = Guid.NewGuid(),
+                OrderStatus = new OrderStatus() {OrderStatusId = 1, Name = "Submitted"}
+            };
+        }
+
+        internal static OrdersBuilder Create() => new OrdersBuilder();
+
+        internal OrdersBuilder WithOrderId(string orderId)
+        {
+            _order.OrderId = orderId;
+            return this;
+        }
+
+        internal OrdersBuilder WithDescription(string description)
+        {
+            _order.Description = description;
+            return this;
+        }
+
+        internal OrdersBuilder WithOrganisationId(Guid organisationId)
+        {
+            _order.OrganisationId = organisationId;
+            return this;
+        }
+
+        internal OrdersBuilder WithCreated(DateTime created)
+        {
+            _order.Created = created;
+            return this;
+        }
+
+        internal OrdersBuilder WithLastUpdated(DateTime lastUpdated)
+        {
+            _order.LastUpdated = lastUpdated;
+            return this;
+        }
+
+        internal OrdersBuilder WithLastUpdatedBy(Guid lastUpdatedBy)
+        {
+            _order.LastUpdatedBy = lastUpdatedBy;
+            return this;
+        }
+
+        internal OrdersBuilder WithOrderStatus(OrderStatus orderStatus)
+        {
+            _order.OrderStatus = orderStatus;
+            return this;
+        }

```

Comment:
Should we be adding these builder options when we're not using them? 

I'd suggest getting rid of them and only adding them once we actually need them.

What do you think?

Reply:
I think its always good to have them, if you are wanting to change the tests later on ect

Reply:
I don't agree, I think it adds noise to your code base, however, it's up to you 😄